regimes/pl/extensions.go

```diff
@@ -34,34 +34,43 @@ var extensionKeys = []*cbc.KeyDefinition{
 		},
 	},
 	{
-		Key: ExtKeyKSeFVATZero,
+		Key: ExtKeyKSeFVATRegion,
```

Comment:
Could this not be determined from the customer's country?

Reply:
Probably in most cases we could, but KSeF allows it not to be the case because it is more related to where the goods or services are provided than to who they are provided to.